spec/system/task_spec.rb

```diff
@@ -1,27 +1,29 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+
+  let(:project){FactoryBot.create(:project)}
+  let(:task){FactoryBot.create(:task)}
```

Comment:
`FactoryBot.`の記載は設定ファイルに追記することで省略できます。
```suggestion
  let(:task){ create(:task) }
```

Reply:
5928d9bでFactoryBot.の記載を削除しました。

Reply:
5928d9b でFactoryBot.の記載を削除しました。